site-src/concepts/api-overview.md

```diff
@@ -4,16 +4,13 @@ This document provides an overview of Gateway API.
 
 ## Roles and personas
 
-There are 3 primary roles in Gateway API:
+There are 3 primary roles in Gateway API, as described in [roles and personas]:
 
-- Infrastructure Provider
-- Cluster Operator
-- Application Developer
+- **Ian** (he/him): Infrastructure Provider
+- **Charlie** (they/them): Cluster Operator
```

Comment:
@candita had the great suggestion to use "Chan" here, which is less western, but still gender-neutral. If we agree here, I'll update #2128 as well.

Reply:
I really like the idea of bringing some less Western names in here. 🙂 At the same time, adopting names from languages and cultures that aren't our own has, historically, been fraught with peril for us Westerners (doubly so when Romanization rears its ugly head), so I'd like to be careful here.

As a given name, Chan comes from Khmer or Sanskrit. I'm just familiar enough with those two to be very concerned indeed that we will not treat that name correctly. 😐 Suppose we swap Charlie for Chihiro instead? That's a Japanese unisex name with a definitive Romanization, and it's likely to be pronounced (at least mostly) correctly by Anglophones when they see it. (And I was able to doublecheck all that with a friend from Japan who teaches Japanese for a living. 🙂)

With Chihiro, we'll have names from Spanish, Irish Gaelic, and Japanese. Hopefully that'll be a reasonable degree of diversity that won't get us into trouble for cultural misappropriation. 🤞🙂

Reply:
It's fine with me.  I just hadn't realized that Charlie was a unisex name, and first suggested Chris.  Later suggested Chan because it wasn't directly related to Christianity and had a more global meaning.

Reply:
Great! I'll go ahead and swap this one; @youngnick, if you can tackle #2128 that'd rock.

Thanks for the suggestion, @candita! 🙂